app/package.json

```diff
@@ -23,8 +23,8 @@
   "homepage": "https://github.com/bcgov/bc-wallet-mobile#readme",
   "scripts": {
     "android": "react-native run-android",
-    "ios": "react-native run-ios --scheme AriesBifold",
-    "start": "react-native start --reset-cache --experimental-debugger",
```

Comment:
These two flags didn't affect the commands, as the debugger isn't enabled yet and the scheme is applied automatically